plugins/woocommerce/client/admin/client/launch-your-store/settings/slotfill.js

```diff
@@ -172,7 +178,9 @@ const SiteVisibility = () => {
 								{
 									a: createElement( 'a', {
 										target: '_blank',
-										href: COMING_SOON_PAGE_EDITOR_LINK,
+										href: getAdminLink(
+											`site-editor.php?postType=wp_template&postId=${ comingSoonTemplateId }&canvas=edit`
```

Comment:
What do you think about linking to the editor initially, and then updating the link once we have the comingSoonTemplateId value?

Right now, the 'You can customize your Coming Soon landing page via the Editor' link appears with a delay of about a second.


https://github.com/user-attachments/assets/06c71ab9-a648-44f8-bfc0-f2263fba5ee9

Reply:
Good idea! I'll make the changes.

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/56743/commits/e43be6643891f4d4358f35fe116a419c7e8438d5

Reply:
@chihsuan LGTM!